src/SignalR/server/Core/src/Internal/DefaultHubDispatcher.cs

```diff
@@ -74,12 +72,10 @@ internal partial class DefaultHubDispatcher<THub> : HubDispatcher<THub> where TH
 
         public override async Task OnConnectedAsync(HubConnectionContext connection)
         {
-            IServiceScope scope = null;
+            IServiceScope scope = _serviceScopeFactory.CreateScope();
```

Comment:
```suggestion
            var scope = _serviceScopeFactory.CreateScope();
```

![image](https://user-images.githubusercontent.com/303201/105560534-e7b83580-5d78-11eb-94f4-5699dc9f20a1.png)